src/nidmm/examples/nidmm_multi_point_measurement.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
+
+import argparse
+import nidmm
+import sys
+
+parser = argparse.ArgumentParser(description='Performs a multi point measurement using the NI-DMM API.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
multi-point I think is the accepted way of spelling. Might be multipoint. It is definitely not multi point. Check.

Reply:
Help shows "multipoint". Fixed Also updated example name